docs/asyncapi.yaml

```diff
@@ -0,0 +1,205 @@
+asyncapi: 2.4.0
+info:
+  title: FFC Payment Statement Generator
+  version: '1.0.0'
+  description: Generate PDF payment statements
+channels:
+  payment/request:
+    subscribe:
+      message:
+        $ref: '#/components/messages/PaymentStatement'
+components:
+  messages:
+    PaymentStatement:
+      name: Payment statement
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/PaymentStatement'
+  schemas:
+    PaymentStatement:
+      type: object
+      required:
+      - businessName
+      - sbi
+      - frn
+      - address
+      - scheme
+      - payments
+      - funding
+      properties:
+        businessName:
+          $ref: '#/components/schemas/BusinessName'
+        sbi:
+          $ref: '#/components/schemas/SBI'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        address:
+          $ref: '#/components/schemas/Address'
+        scheme:
+          $ref: '#/components/schemas/Scheme'
+        payments:
+          type: array
+          items:
+            $ref: '#/components/schemas/Payment'
+        funding:
+          type: array
+          items:
+            $ref: '#/components/schemas/Funding'
+    BusinessName:
+      type: string
+      description: The business name
+      example: 'FFC Ltd'
+    SBI:
+      type: number
+      description: Single Business Identifier
+      minimum: 105000000
+      maximum: 999999999
+      example: 123456789
+    FRN:
+      type: number
+      description: Firm Reference Number
+      minimum: 1000000000
+      maximum: 9999999999
+      example: 1234567890
+    Address:
+      type: object
+      required:
+      - postcode
+      properties:
+        line1:
+          type: string
+          description: The first line of the address
+          example: '1 The Street'
+        line2:
+          type: string
+          description: The second line of the address
+          example: 'The Area'
+        line3:
+          type: string
+          description: The third line of the address
+          example: 'The Town'
+        line4:
+          type: string
+          description: The fourth line of the address
+          example: 'The Region'
```

Comment:
standardise using single quotes or dbl

Reply:
actually, looking at yer golden standard `async` doc it doesn't use quotes at all: https://github.com/DEFRA/ffc-pay-enrichment/blob/main/docs/asyncapi.yaml

Reply:
I blame co-pilot haha